pkg/controller/upgradeconfig/upgradeconfig_controller_test.go

```diff
@@ -192,12 +241,32 @@ var _ = Describe("UpgradeConfigController", func() {
 
 				Context("When the cluster is ready to upgrade", func() {
 					Context("When a cluster upgrade client can't be built", func() {
+						var clusterVersionList *configv1.ClusterVersionList
+						BeforeEach(func() {
+							clusterVersionList = &configv1.ClusterVersionList{
```

Comment:
Minor nit, looks like the `clusterVersionlist` variable could all be moved to one of the outer (even the outermost) BeforeEach() contexts, that would tidy things up a bit..

Reply:
Yeah I think that kind of work can be done in a refactor? I would presume we are duplicating vars else where in this file.

Reply:
Yeah, the overall refactor will be a good place to just take a fresh look at things and re-structure to a common approach. Agree - don't need to do anything on this one here.  Refs OSD-4590